app/components/info-cards/song-card.tsx

```
@@ -0,0 +1,32 @@
+    // if (!hasCookie('session')) {
+    //   redirectToLogin(window.location.pathname);
+    //   return;
+    // }
+
+    if (currentSong?.id === props.songID && isPlaying) {
+      pauseSong();
+      return;
+    }
+
+    const song = {
+      id: props.songID!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)